### PR TITLE
Added a disclaimer regarding rm -rf command

### PR DESCRIPTION
**Pull request by coderxx**
A lot of people who jump to this section might end up deleting data without realising what they're doing. I feel it's proper to include a disclaimer.

### CODE REVIEW

File: pages/03.Troubleshooting/docs.en.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ Even through the effort of the dev and test teams, it might happen. At first, le
 
 There are several ways to do that. The easiest is to go to the `/app/cache` folder and delete its content. If you want to do it via CLI command, navigate to the Mautic root folder and run `rm -rf app/cache/*`. The new cache files will generate itself after the next Mautic refresh in the browser.
 
+#### (Warning: Improper misuse of the 'rm -rf' command can delete entire folders and files from your project. Don't try it if you don't know what you're doing.)
```
Comment (RCheesley) [inline pages/03.Troubleshooting/docs.en.md:29]: Let's use the warning box markup to make it stand out

```suggestion
>>>> Warning: Improper misuse of the 'rm -rf' command can delete entire folders and files from your project. Don't try it if you don't know what you're doing.
```

File: pages/03.Troubleshooting/docs.en.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ Even through the effort of the dev and test teams, it might happen. At first, le
 
 There are several ways to do that. The easiest is to go to the `/app/cache` folder and delete its content. If you want to do it via CLI command, navigate to the Mautic root folder and run `rm -rf app/cache/*`. The new cache files will generate itself after the next Mautic refresh in the browser.
```
Comment (RCheesley) [inline pages/03.Troubleshooting/docs.en.md:27]: Ooops, this is an old reference, let's update it while we are here!

```suggestion
There are several ways to do that. The easiest is to go to the `/var/cache` folder and delete its content. If you want to do it via CLI command, navigate to the Mautic root folder and run `rm -rf var/cache/*`. The new cache files will generate itself after the next Mautic refresh in the browser.
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Good suggestion - I noticed an outdated reference to app/cache which I suggested we fix while we're here, and also think we should use the markup to create a warning box to make it visually stand out.

Thank you for taking the time to make the contribution!

**Review by coderxx — COMMENTED** [review]
Updated the cache location. Didn't recognise this outdated reference. 
Changed message to warning

**Review by coderxx — COMMENTED** [review]
Fixed reference to cache folder

**Review by coderxx — COMMENTED** [review]
I think it looks good. No more changes that I can see.

**Review by RCheesley — APPROVED** [review]
🚀  LGTM sorry for the delay @coderxx !

**Pull request merged**